rust/core-lib/src/tabs.rs

```diff
@@ -194,6 +194,8 @@ impl CoreState {
         let plugin_paths = self.config_manager.get_plugin_paths();
         self.plugins.reload_from_paths(&plugin_paths);
         let languages = self.plugins.make_languages_map();
+        let languages_ids = languages.named.keys().cloned().collect::<Vec<_>>();
```

Comment:
as mentioned above, this can just be `languages.iter().cloned().collect::<Vec<_>>();`.

Reply:
er, the above is slightly wrong. I would do this as,
```rust
langauges.iter().map(|l| l.name.clone()).collect::<Vec<_>>();
```

Reply:
Great! I’ll fix it as soon as I get back from work!